src/components/inspector/controls/TextControl.tsx

```diff
@@ -20,7 +20,7 @@ const TextControl: React.FC<TextControlPropsType> = ({
   placeholder = '',
 }) => {
   const { setValueFromEvent } = useForm()
-  const value = usePropsSelector(name)
+  let value = usePropsSelector(name)
```

Comment:
why?

Reply:
This was done when I tried JSON stringifying the value, if it was an object, but that didn't work so I'll return it to const